docs/pages/admin-guides/management/operations/ca-rotation.mdx

```diff
@@ -128,54 +147,73 @@ also signs client certificates for users connecting to Windows desktops and
 Teleport SSH servers. Teleport-protected servers and Windows desktops use these
 certificates.
 
-Once you have completed the rotation and reached the final `standby` phase,
+Before you complete the rotation and reach the final `standby` phase,
 users who have signed into Teleport must reauthenticate to receive a user
 certificate from the new CA, otherwise Teleport client commands fail.
 
-If you have registered Windows desktops with Teleport, [follow the
-guide](../../../enroll-resources/desktop-access/active-directory.mdx) to export
-the Teleport user CA so the Windows Desktop Service can authenticate to RDP
-hosts. Verify that you can connect to registered desktops throughout the
-rotation.
+If you have registered Windows desktops with Teleport,
+[follow the guide](../../../enroll-resources/desktop-access/active-directory.mdx)
+to export the Teleport user CA so the Windows Desktop Service can authenticate
+to RDP hosts.
+Verify that you can connect to registered desktops throughout the rotation.
 
 ### `db` and `db_client`
 
 The `db` and `db_client` CAs issue certificates that the Teleport Database
-Service uses to communicate with self-hosted databases. 
+Service uses to communicate with self-hosted databases.
 
 The Teleport Database Service presents a certificate signed by the `db_client`
 CA when communicating with a self-hosted database, which an admin configures to
-trust certificates issued by the CA. 
+trust certificates issued by this CA.
 
 Admins can configure self-hosted databases to present a certificate signed by
 the `db` CA, which the Database Service uses to verify that a database server is
 a genuine Teleport-protected resource. Alternatively, self-hosted databases can
 present a certificate signed by a custom CA, and admins can configure the
 Teleport Database Service to trust the CA.
 
-#### Beginning the rotation
+#### Rotating the database CAs
 
-The Teleport Database Service starts using client certificates issued by the new
-CA to connect to databases at the `update_clients` phase. To avoid losing access
-to your self-hosted databases in the `update_clients` phase, you should
-reconfigure your databases in the `init` phase, then verify that you can still
-access your databases after transitioning to the `update_clients` phase.
+<Admonition type="note">
```

Comment:
Same comment here re: multiple successive Admonitions. I would structure this as regular body text, since at the moment the Admonitions aren't doing any work for the reader.

Reply:
I opted to drop the first and keep the second